doc/architecture.rst

```diff
@@ -90,4 +90,11 @@ You can set in this section:
 - *uri* : the protocol, the host and the port of your LDAP server
 - *base* : the database where users have to be searched
 - *ugroup* : the LDAP group which the users are members
-- *expiration* : the TTL of the generated token
\ No newline at end of file
+- *expiration* : the TTL of the generated token
+
+Slurm-web also takes into account of the Private data parameter from
+`Slurm <https://slurm.schedmd.com/slurm.conf.html>`_. The python wrapper
+edf-hpc/`pyslurm <https://github.com/edf-hpc/pyslurm/>`_ has been
```

Comment:
The fact that PySLURM has been modified is not a useful information for slurm-web users and administrators.